src/transform/tests.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+// This file only contains functions used to test core object mapper functionality
+
+exports.tests = {
+  'test-foo': function(value) {
+    return value + '-foo'
+  },
+  'test-destination': function(val, _src, dst) {
+    dst.manual = val.a + val.b
+  },
+  'test-empty-array': function(sourceValue) {
+    var i
```

Comment:
Not sure if this was exiting or not, but I think we can remove this declaration here and do it in the for loop instead.

Reply:
Lol, that was from the original tests - went straight from eyes to fingers by-passing brain